### PR TITLE
Lab1

### PR DESCRIPTION
**Pull request by Mick-Ross**
@afspicciati

### CODE REVIEW

File: Lab 1/Lab1.ipynb [hunk 1]
```diff
@@ -2,11 +2,539 @@
...
+    "\n",
+    "from scipy import stats"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6f3406c7-4cbf-47bb-8cb8-99c29e565e97",
+   "metadata": {},
+   "source": [
+    "# Statistics Basics\n",
```
Comment (afspicciati) [inline Lab 1/Lab1.ipynb:23]: For section A, provide some background explanation of what a normal distribution/Gaussian is. Providing formulas can be useful as well (take a look at my LateX to get an idea how LateX works).

File: Lab 1/Lab1.ipynb [hunk 2]
```diff
@@ -2,11 +2,539 @@
...
+   "source": [
+    "stats.norm.ppf(stats.norm.cdf(1))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "072ee033-5455-4292-ba81-6b896407cafe",
+   "metadata": {},
+   "source": [
+    "#### 2. Exploring other continuous analytic distributions. (Chi-squared)"
```
Comment (afspicciati) [inline Lab 1/Lab1.ipynb:234]: Again, for part 2A give some background on what the Chi-squared distribution is, and when/why it's relevant.

For part 2B, you may want to show multiple plots of the Chi-squared distribution, to show how changing the degrees of freedom parameter changes the plot.

File: Lab 1/Lab1.ipynb [hunk 3]
```diff
@@ -2,11 +2,539 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7163c1b0-39e7-4c70-bf69-ed30c0ee917f",
+   "metadata": {},
+   "source": [
+    "#### 3B. \n",
+    "Let's propose a scientific question.\n",
+    "\n",
+    "__What is the probability that 3.72 is a measurement of the signal or of a greater signal?__"
```
Comment (afspicciati) [inline Lab 1/Lab1.ipynb]: Explain what is meant by "signal" here. Why is it significant if the measurement is of "greater signal"?

File: Lab 1/Lab1.ipynb [hunk 4]
```diff
@@ -2,11 +2,539 @@
...
+    "prob = 1 - stats.chi2.cdf(3.72, 55, loc = 3., scale = 0.01)\n",
+    "print(prob)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1066bb3e-c57c-4d2a-a788-9e6d4595d832",
+   "metadata": {},
+   "source": [
+    "#### 3E. The equivalent 'sigma' of our probability calculation, in the context of a normal distribution."
```
Comment (afspicciati) [inline Lab 1/Lab1.ipynb]: Explain what the relevance of the Sigma rating is

File: Lab 1/Lab1.ipynb [hunk 5]
```diff
@@ -2,11 +2,539 @@
...
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "x = np.linspace(0,40,41)\n",
+    "pmf = stats.binom.pmf\n",
+    "plt.plot(x, pmf(x, n = 40, p = 0.5))\n",
```
Comment (afspicciati) [inline Lab 1/Lab1.ipynb]: Try using plt.stairs() or plt.step() here to show more clearly that this is a plot of a discrete variable. You could also use plt.vlines([x-locations], ymin, ymax) I.E. plt.vlines(x, 0, pmf).

Do this for the next plot as well.

File: Lab 1/Lab1.ipynb [hunk 6]
```diff
@@ -2,11 +2,539 @@
...
+    "plt.plot(x,pmf2)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0ea73d79-b9c9-48c0-944d-c508ed70342c",
+   "metadata": {},
+   "source": [
+    "When the probability is away from 0.5, however, we begin to see a skew in the distribution. The peak of the distribution nears the expected value of successful events."
```
Comment (afspicciati) [inline Lab 1/Lab1.ipynb:534]: Good explanation!

File: Lab 1/Lab1.ipynb [hunk 7]
```diff
@@ -2,11 +2,539 @@
...
+   "source": [
+    "#### 3D."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3f7bd076-e1d5-421a-9685-cfd357c4d9d7",
+   "metadata": {},
+   "source": [
+    "Probabilities and sigmas are also discrete in these non-continuous distributions. So compared to continuous probability distributions, instead of thinking in terms of integrals and infinitesimals, we must think of actually summing the probabilities of these discrete values. The biggest catch might be, unlike continuous distributions, we can also calculate the probability at a single value of k."
```
Comment (afspicciati) [inline Lab 1/Lab1.ipynb:593]: I really like the point you made in your last sentence; I'm going to have to add that to mine :P

**Review by afspicciati — COMMENTED** [review]
I like your explanations, they are well written. Main points are just to make sure you add a summary of what the normal and Chi-squared distributions are for section 1A and 2A; and to use a step function for your discrete distributions. I might also suggest adding titles and axis titles for your plots (see my code for and example of how to do this).

**Comment by Mick-Ross** [comment]
@hrzhao76 
Finished updating lab according to feedback.

**Pull request merged**